tests/acceptance/features/apiAntivirus/antivirus.feature

```diff
@@ -127,3 +127,35 @@ Feature: antivirus
       | old              |
       | new              |
       | spaces           |
+
+  @env-config
+  Scenario Outline: upload a file with virus with various infected file handling setting set to continue
+    Given the config "ANTIVIRUS_INFECTED_FILE_HANDLING" has been set to "continue"
+    And using <dav-path-version> DAV path
+    When user "Alice" uploads file "filesForUpload/filesWithVirus/eicar.com" to "/aFileWithVirus.txt" using the WebDAV API
+    Then the HTTP status code should be "201"
+    And user "Alice" should get a notification with subject "Virus found" and message:
+      | message                                                                             |
+      | Virus found in aFileWithVirus.txt. Upload not possible. Virus: Win.Test.EICAR_HDB-1 |
+    And as "Alice" file "/aFileWithVirus.txt" should exist
```

Comment:
The notification message says `Upload not possible`, but the file is uploaded. What might the expected message be?

Reply:
there's a bug report for this https://github.com/owncloud/ocis/issues/6494, need someone from owncloud to verify

Reply:
@ScharfViktor we have a misleading notification when, `ANTIVIRUS_INFECTED_FILE_HANDLING` is set to `continue` and a file with the virus is uploaded. There's an issue report https://github.com/owncloud/ocis/issues/6494 however what to do in this scenario? Do we not check the notification? or change the message to something like `Virus found in aFileWithVirus.txt. Upload successful. Virus: Win.Test.EICAR_HDB-1` and add the test in expected to fail or wait till the bug is fixed?